src/library/dao/cue.cpp

```diff
@@ -19,20 +21,22 @@ Cue::Cue(TrackId trackId)
           m_iPosition(-1),
           m_iLength(0),
           m_iHotCue(-1),
-          m_label("") {
+          m_label(""),
```

Comment:
The default constructor that is invoked implicitly does exactly this. We don't need to explicitly copy an empty C-style string.